tests/ha/control_pod/test_control_pod_restart.py

```diff
@@ -364,49 +364,55 @@ def test_failover_control_pod_kubectl(self):
                                                     log_prefix=self.test_prefix, skipcleanup=True)
         assert_utils.assert_true(resp[0], resp[1])
         LOGGER.info("Step 1: Performed WRITEs-READs-Verify with variable sizes objects.")
-
         LOGGER.info("Control pod %s is hosted on %s node", self.control_pod_name, self.control_node)
-
-        failover_node = self.system_random.choice([ele for ele in self.host_worker_list if ele !=
-                                                   self.control_node])
-        LOGGER.debug("Fail over node is: %s", failover_node)
-
-        LOGGER.info("Step 2: Failover control pod %s to node %s and check cluster status",
-                    self.control_pod_name, failover_node)
-        pod_yaml = {self.control_pod_name: self.modified_yaml}
-        resp = self.ha_obj.failover_pod(pod_obj=self.node_master_list[0], pod_yaml=pod_yaml,
-                                        failover_node=failover_node)
+        LOGGER.info("Step 2: Taint the control node %s and delete control pod %s",
+                    self.control_node, self.control_pod_name)
+        self.node_master_list[0].execute_cmd(cmd=cmd.K8S_TAINT_CTRL.format(self.control_node))
+        LOGGER.info("Restart the control pod by kubectl delete.")
+        resp = self.node_master_list[0].delete_pod(pod_name=self.control_pod_name, force=True)
+        LOGGER.debug("Response: %s", resp)
+        assert_utils.assert_true(resp[0], f"Failed to delete pod {self.control_pod_name} by "
+                                          "kubectl delete")
+        LOGGER.info("Step 2: Tainted the control node %s and deleted control pod %s",
+                    self.control_node, self.control_pod_name)
+        LOGGER.info("Sep 3: Check cluster status and new control node hosting control pod")
+        resp = self.ha_obj.poll_cluster_status(self.node_master_list[0], timeout=60)
```

Comment:
Is 60 sec enough? Can we use it from common config file if possible ?

Reply:
Should be enough, let me take from config though